pom.xml

```diff
@@ -549,11 +453,6 @@
                <version>${ear.plugin.version}</version>
             </plugin>
             <plugin>
-               <groupId>com.pyx4j</groupId>
-               <artifactId>maven-junction-plugin</artifactId>
```

Comment:
Should allow to create symbolic links on Win/*nix systems. Retired, we don't use that anywhere.